packages/forms/src/text/FormikCurrencyField.tsx

```diff
@@ -7,6 +7,8 @@ function formatInputError(error: string): ReactNode {
   return error || undefined;
 }
 
+export const EMPTY_ERROR_MESSAGE = 'EMPTY_ERROR_MESSAGE';
```

Comment:
Will we create for every field component its own `EMPTY_ERROR_MESSAGE` constant? Or we will move this one onto a common folder and reuse it?

Reply:
@timurhamitov I think we'll have one common constant. Let's do it on the next PR.